docs/source/scenarios/container_inventory_management.rst

```diff
@@ -630,14 +630,104 @@ random mode, we hope this could help you learn how to use the environment interf
+
+CIM Inter Epoch Data
+''''''''''''''''''''
+
+Generally speaking, the number of epochs in this scene will be greater
+than 100. In order to facilitate users to select specific data and
+observe the overall or partial data trend, the visualization tool
+provides data selection options in two dimensions.
+
+To change "Start Epoch" and "End Epoch", user could specify the selected
+data range. To change "Epoch Sampling Ratio", user could change the
+sampling rate of selected data, similar as `Citi\_Bike Detail
+Data <#Citi_Bike%20Detail%20Data>`__.
+
+For description of Attributes Selection in charts and Formula
+Calculation, please refer to `Citi\_Bike Detail
+Data <#Citi_Bike%20Detail%20Data>`__.
+
+.. figure:: ..\images\visualization\dashboard\cim_extro_epoch.gif
+   :alt: cim\_extro\_epoch
+
+   cim\_extro\_epoch
+CIM Intra Epoch Data
+''''''''''''''''''''
+
+This part shows the data under a selected epoch. By scrolling the
+slider, users can select different epochs. Furthermore, this part of
+data is divided into two dimensions: by snapshot and by port according
+to time and space. In terms of data display, according to the different
+types of attributes, it is divided into two levels: acc data
+(accumulated attributes. e.g. acc\_fulfillment) and detail data.
+
+If user choose to view information by ports, attributes of the selected
+port would be displayed.
+
+Chart characteristics and data selection method please refer to
+`Citi\_Bike Detail Data <#Citi_Bike%20Detail%20Data>`__.
+
+.. figure:: ..\images\visualization\dashboard\cim_intra_epoch_by_ports.gif
+   :alt: cim\_intra\_epoch\_by\_ports
+
+   cim\_intra\_epoch\_by\_ports
+If user choose to view data by snapshots, attributes of selected
+snapshot would be displayed. The charts and data involved in this part
+are relatively rich, and we will introduce them by level.
+
+**Acc Data**
+
+This part includes the transfer volume hot map, bar chart of port
+accumulated attributes and top-5 ports of different attributes.
+
+As shown in the following example, the x-axis and y-axis of transfer
+volume hot map refers to destinate port index and start port index
+respectively. The rect refers to the volume of cargoes transfer from
+start port to destinate port. By changing the snapshot index, user could
+view the dynamic changes in the volume of cargo delivered by the port
+over time in the current epoch.
+
+The bar chart of Port Acc Attributes displays the global change of
+ports.
+
+.. figure:: ..\images\visualization\dashboard\cim_intra_epoch_by_snapshot_acc_data.gif
+   :alt: cim\_intra\_epoch\_by\_snapshot\_acc\_data
+
+   cim\_intra\_epoch\_by\_snapshot\_acc\_data
+**Detail Data**
+
+Since the cargoes is transported through vessels, information of vessels
+could be viewed by snapshot.
+
+Same as ports, user could change the sampling rate of vessels.
+
+Bar chart of Port Detail Attributes and Formula Calculation please refer
+to `Citi\_Bike Detail Data <#Citi_Bike%20Detail%20Data>`__.
```

Comment:
xxx_yyy

Reply:
ref style: `k8s mode <../distributed_training/orchestration_with_k8s.html#orchestration-with-k8s>`_

Reply:
local build for testing